xml/ha_hawk2_manage_i.xml

```diff
@@ -0,0 +1,365 @@
+      </step>
+     </substeps>
+    </step>
+   </procedure>
+  </sect2>
+
+  <sect2 xml:id="sec.conf.hawk2.manage.migrate">
+   <title>Migrating Cluster Resources</title>
+   <para> As mentioned in <xref linkend="sec.conf.hawk2.failover"/>, the cluster will fail over
+    (migrate) resources automatically in case of software or hardware failures&mdash;according
+    to certain parameters you can define (for example, migration threshold or resource stickiness).
+    Apart from that, you can manually migrate a resource to another node in the cluster. Or you
+    decide to move it away from the current node and leave the decision about where to put it to the
+    cluster. </para>
+   <procedure xml:id="pro.hawk2.rsc.migrate">
+    <title>Manually Migrating a Resource</title>
+    <step>
+     <para>Log in to &hawk;2:</para>
+     <screen>https://<replaceable>HAWKSERVER</replaceable>:7630/</screen>
+    </step>
+    <step>
+     <para> In the left navigation bar, select <guimenu>Status</guimenu>. The list of
+       <guimenu>Resources</guimenu> also shows the <guimenu>Status</guimenu>. </para>
+    </step>
+    <step>
+     <para>In the list of <guimenu>Resources</guimenu>, go to the respective resource.</para>
+    </step>
+    <step>
+     <para><remark>taroth 2015-12-07: krig, is there any way to select which node to migrate the
+       resource to? krig: also, Hawk1 has two options: "Move" (lets you select where to move the rsc
+       to and creates a location constraint with an INFINITY score for the destination node) and
+       "Away from current node" (creates a location constraint with a -INFINITY score for the
+       current node) - si there anything like this with Hawk2? </remark>In the
+       <guimenu>Operations</guimenu> column click the arrow down button and select
+       <guimenu>Migrate</guimenu>. To continue, confirm the message that appears.</para>
+    </step>
+   </procedure>
+   <para> To allow a resource to move back again, proceed as follows: </para>
+   <procedure xml:id="pro.hawk2.rsc.migrate.back">
+    <title>Unmigrating a Resource</title>
+
+    <step>
+     <para>Log in to &hawk;2:</para>
+     <screen>https://<replaceable>HAWKSERVER</replaceable>:7630/</screen>
+    </step>
+    <step>
+     <para> In the left navigation bar, select <guimenu>Status</guimenu>. The list of
+       <guimenu>Resources</guimenu> also shows the <guimenu>Status</guimenu>. </para>
+    </step>
+    <step>
+     <para>In the list of <guimenu>Resources</guimenu>, go to the respective resource.</para>
+    </step>
+    <step>
+     <para>In the <guimenu>Operations</guimenu> column click the arrow down button and select
+       <guimenu>Unmigrate</guimenu>. To continue, confirm the message that appears.</para>
+     <para>
+      <remark>taroth 2015-12-07: krig, is the following still correct? and which crm shell command
+       is used? the "Command Log" unfortunately did not show anything...</remark> The resource
```

Comment:
Yes, still correct. These commands don't actually call `crm` but use the pacemaker `crm_resource` command directly, but yes, we should probably log them in the command log too.